.github/workflows/therock-test-packages-multi-node.yml

```
@@ -52,6 +52,14 @@ jobs:
 
       - name: Test
         run: |
+          scancel -p 32nodexgmi36 --reservation=ci_cd_test <<'EOF'
+          salloc -N 4 --reservation=ci_cd_test --exclusive bash <<'EOF'
           source /home/arravikum/TheRock/.venv/bin/activate
           cd /home/arravikum/cvs
-          pytest -vvv --log-file=/tmp/rccl_log.log -s ./tests/rccl/rccl_multinode_cvs.py --cluster_file ./input/cluster.json --config_file ./input/mi300_config.json --html=/var/www/html/cvs/ci_test_report.html --capture=tee-sys --self-contained-html
+
+          pytest -vvv -s ./tests/rccl/rccl_multinode_cvs.py \
+              --cluster_file ./input/cluster.json \
+              --config_file ./input/mi350_config.json \
+              --log-file=/tmp/rccl_log.log \
+              --html=/home/arravikum/cvs/ci_test_report.html --capture=tee-sys --self-contained-html
```

can we add new line characters here just like above?

```suggestion
              --html=/home/arravikum/cvs/ci_test_report.html \
              --capture=tee-sys \
              --self-contained-html
```